run-spartacus-ccv2-e2es.sh

```diff
@@ -4,17 +4,15 @@
 # export ENDPOINT_URL_PUBLIC_SPARTACUS=https://spartacusstore.cg79x9wuu9-eccommerc1-p1-public.model-t.myhybris.cloud
 # export ENDPOINT_URL_PUBLIC_API=https://api.cg79x9wuu9-eccommerc1-p1-public.model-t.myhybris.cloud
 
-CYPRESS_ROOT_FOLDER="cypress/integration"
+CYPRESS_ROOT_FOLDER="cypress/e2e"
 CYPRESS_B2C_FOLDER="$CYPRESS_ROOT_FOLDER/regression"
 CYPRESS_B2B_FOLDER="$CYPRESS_ROOT_FOLDER/b2b/regression"
 CYPRESS_ACCESSIBILITY_FOLDER="$CYPRESS_ROOT_FOLDER/accessibility"
 CYPRESS_SSR_FOLDER="$CYPRESS_ROOT_FOLDER/ssr"
 CYPRESS_VENDOR_FOLDER="$CYPRESS_ROOT_FOLDER/vendor"
 
 
-export E2ES_TO_RUN="$CYPRESS_B2C_FOLDER/checkout/checkout-flow.core-e2e-spec.ts,
-$CYPRESS_B2C_FOLDER/homepage/homepage.core-e2e-spec.ts
-"
+export E2ES_TO_RUN="$CYPRESS_B2C_FOLDER/checkout/checkout-flow.core-e2e.cy.ts,$CYPRESS_B2C_FOLDER/homepage/homepage.core-e2e.cy.ts"
```

Comment:
In addition of filename adjustment, the E2ES_TO_RUN value needs to be on a single line, otherwise cypress found 176 tests instead of 2.